app/src/androidTest/java/org/mozilla/fenix/ui/robots/TabsRobot.kt

```diff
@@ -20,4 +24,8 @@ class TabsRobot {
     }
 }
 
-// Locaters here
+private fun tabsCounter() = Espresso.onView(ViewMatchers.withId(R.id.counter_box))
```

Comment:
Why do we need to do `Espresso.onView` here instead of just `onView`?

Reply:
agreed, `onView` is all you need :)

Reply:
TabsRobot file is no longer needed. There is BrowserRobot now which is the same thing.